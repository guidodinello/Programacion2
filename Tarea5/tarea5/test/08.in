# Prueba de unión, diferencia e intersección entre conjuntos vacíos y singleton 
# vacío - vacío
unionDeConjuntos 0
iteradorDeConjunto
diferenciaDeConjuntos 0
iteradorDeConjunto
interseccionDeConjuntos 0
iteradorDeConjunto
# vacío - singleton
diferenciaDeConjuntos 1 3
iteradorDeConjunto
interseccionDeConjuntos 1 4
iteradorDeConjunto
unionDeConjuntos 1 5
iteradorDeConjunto
# singleton - vacío 
unionDeConjuntos 0
iteradorDeConjunto
diferenciaDeConjuntos 0
iteradorDeConjunto
interseccionDeConjuntos 0
iteradorDeConjunto
# para volver a tener un singleton
unionDeConjuntos 1 6
# singleton - singleton
unionDeConjuntos 1 6
iteradorDeConjunto
diferenciaDeConjuntos 1 3
iteradorDeConjunto
interseccionDeConjuntos 1 6
iteradorDeConjunto
diferenciaDeConjuntos 1 6
iteradorDeConjunto
unionDeConjuntos 1 6
interseccionDeConjuntos 1 3
iteradorDeConjunto
unionDeConjuntos 1 6
unionDeConjuntos 1 4
iteradorDeConjunto
# para volver a tener un singleton
interseccionDeConjuntos 1 6
iteradorDeConjunto
diferenciaDeConjuntos 1 8
iteradorDeConjunto
interseccionDeConjuntos 1 8
iteradorDeConjunto
# para volver a tener un singleton
unionDeConjuntos 1 6
unionDeConjuntos 1 8
iteradorDeConjunto
Fin
