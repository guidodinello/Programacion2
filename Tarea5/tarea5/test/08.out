1>#  Prueba de unión, diferencia e intersección entre conjuntos vacíos y singleton .
2>#  vacío - vacío.
3>Union.
4>No hay elementos en el conjunto.
5>Diferencia.
6>No hay elementos en el conjunto.
7>Intersección.
8>No hay elementos en el conjunto.
9>#  vacío - singleton.
10>Diferencia.
11>No hay elementos en el conjunto.
12>Intersección.
13>No hay elementos en el conjunto.
14>Union.
15>5 
16>#  singleton - vacío .
17>Union.
18>5 
19>Diferencia.
20>5 
21>Intersección.
22>No hay elementos en el conjunto.
23>#  para volver a tener un singleton.
24>Union.
25>#  singleton - singleton.
26>Union.
27>6 
28>Diferencia.
29>6 
30>Intersección.
31>6 
32>Diferencia.
33>No hay elementos en el conjunto.
34>Union.
35>Intersección.
36>No hay elementos en el conjunto.
37>Union.
38>Union.
39>4 6 
40>#  para volver a tener un singleton.
41>Intersección.
42>6 
43>Diferencia.
44>6 
45>Intersección.
46>No hay elementos en el conjunto.
47>#  para volver a tener un singleton.
48>Union.
49>Union.
50>6 8 
51>Fin.
